spec/tapioca/gem/pipeline_spec.rb

```diff
@@ -3898,6 +3898,12 @@ def foo; end
         end
       RB
 
+      sorbet_initialize = if RUBY_VERSION >= "3.1"
```

Comment:
You can use erb instead:

```
        <% if ruby_version(">= 3.1") %>
          def initialize(*args, **_arg1, &blk); end
        <% else %>
          def initialize(*args, &blk); end
        <% end %>
```

See [here](https://github.com/Shopify/tapioca/blob/main/spec/tapioca/gem/pipeline_spec.rb#L162-L166) for example.